content/en/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md

```diff
@@ -98,6 +98,11 @@ read-write by a single Node. It defines the [StorageClass name](/docs/concepts/s
 `manual` for the PersistentVolume, which will be used to bind
 PersistentVolumeClaim requests to this PersistentVolume.
 
+{{< note >}}
+For this example we use the `ReadWriteOnce` access mode. For production use we
+recommend `ReadWriteOncePod`.
```

Comment:
We [avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we) in docs.
```suggestion
This example use the `ReadWriteOnce` access mode, for simplicity. For production use, the Kubernetes
project recommends using the `ReadWriteOncePod` access mode instead.
```

Reply:
Thank you for the link!